libraries/rush-lib/src/logic/operations/ShellOperationRunner.ts

```diff
@@ -75,6 +79,11 @@ export class ShellOperationRunner implements IOperationRunner {
         // Run the operation
         terminal.writeLine(`Invoking: ${this.commandToRun}`);
 
+        // Log any ignored parameters in verbose mode
+        if (this._ignoredParameterValues.length > 0) {
+          terminal.writeVerboseLine(`Ignored parameters: ${this._ignoredParameterValues.join(' ')}`);
```

Comment:
Log this before the invocation line. Make clear that these were ignored by project-level configuration. Log it with regular `writeLine`, since it is equally important.